### PR TITLE
Add back padding on mobile hero for DP sale

### PR DESCRIPTION
**Pull request by walaura**
## Why are you doing this?
it looks wrong right now in prod

This is the fixed one:
screenshot 2018-10-25 at 3 56 17 pm

### CODE REVIEW

**Comment by LATaylor-guardian** [comment]
👍

**Comment by LATaylor-guardian** [comment]
(I can't approve it for some reason)

File: assets/components/featuredDigitalPack/flashSaleDigitalPack.scss [hunk 1]
```diff
@@ -31,6 +31,7 @@
   .component-flash-sale-featured-digital-pack__countdownbox {
     border-left: 1px solid gu-colour(garnett-neutral-4);
     padding-left: 5px;
+    padding-bottom: 70px;
```
Comment (Ap0c) [inline assets/components/featuredDigitalPack/flashSaleDigitalPack.scss]: Can you make this `120px`? It's clipping the CTA on the narrower breakpoints.

Comment (walaura) [reply assets/components/featuredDigitalPack/flashSaleDigitalPack.scss]: done

Comment (walaura) [reply assets/components/featuredDigitalPack/flashSaleDigitalPack.scss]: i thought the clipping was intentional however? provided the image goes under the button

**Pull request merged**